datasets/fhad/make_data.py

```diff
@@ -10,8 +10,16 @@
 # Set Path here
 # EXAMPLE:
 #   ROOT = '/path/to/FHAD/hand_pose_action'
+import os
+import sys
 
-ROOT = None
+ROOT = "/mnt/batch/tasks/shared/LS_root/mounts/clusters/wack/code/Users/aengus.crowley/FHAD/First-Person Hand Action Benchmark (F-PHAB)"
```

Comment:
This isn't your fault but you shouldn't commit personal paths like this.   The code is bad and it should take this as a CLI argument instead of having to edit the file.   I can add that functionality but let's get this PR cleaned up first.  I don't want to make a conflict on the main branch.